src/sentry/static/sentry/app/components/group/issuePluginActions.jsx

```diff
@@ -77,106 +98,88 @@ const IssuePlugin = React.createClass({
       state.error = {message: t('An unknown error occurred.')};
     }
     this.setState(state);
-  },
+  }
 
   fetchData() {
-    this.setState({
-      loading: true
+    this.onLoad(() => {
```

Comment:
will everything use onLoad? If so maybe we just have settings call onLoad automatically (vs wiring up fetchData)